tools/lib/provision.py

```diff
@@ -162,6 +162,8 @@
         "postgresql-10-pgroonga",
         "postgresql-10-tsearch-extras",
         "virtualenv",  # see comment on stretch
+        # Required by phantomjs. Installed by default on other versions.
+        "libfontconfig1",
```

Comment:
I think we should add it on trusty/xenial/stretch too; I think in a docker environment (or similar) it might not actually end up installed on other versions.  There's no cost to it being there, and it'll help avoid problems.

Reply:
I guess that means adding it to `UBUNTU_COMMON_APT_DEPENDENCIES`

Reply:
Done.